### PR TITLE
Handle .

### PR DESCRIPTION
**Pull request by unparalleled-js**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
None

## Description
Handle the case when using the sdk unify command, that you are in the directory you want to unify and you pass it the input ".". Before my changes, doing this causes the file to be named `integration-..yml` by default, instead of the name of the directory you are in, for example `integration-Code42.yml`.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ronykoz** [comment]
@unparalleled-js thanks for the contribution, @reutshal will take a look at it.

File: demisto_sdk/commands/unify/unifier.py [hunk 1]
```diff
@@ -26,7 +26,10 @@ class Unifier:
     def __init__(self, input: str, dir_name=INTEGRATIONS_DIR, output: str = '',
                  image_prefix=DEFAULT_IMAGE_PREFIX, force: bool = False):
 
-        directory_name = ""
+        directory_name = ''
+        # Changing relative path to current abspath fixed problem with default output file name.
+        if input == '.' and output == '':
```
Comment (reutshal) [inline demisto_sdk/commands/unify/unifier.py:31]:
```suggestion
        if input == '.':
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,7 @@
 * Fixed an issue where **validate** command checked docker when running on all files.
 * Fixed an issue where **validate** command did not fail when docker image was not on the latest numeric tag.
 * Fixed an issue where beta integrations were not validated correctly in **validate** command.
+* Fixed an issue where the default outfile file name for the **unify** command would be anomalous when using an input file path of ".".
```
Comment (reutshal) [inline CHANGELOG.md:28]: Please move this note to line 2 under `changelog` headline

File: demisto_sdk/commands/unify/unifier.py (hunk 1 above)
Comment (reutshal) [reply demisto_sdk/commands/unify/unifier.py:31]: Even when the output exists we want to do this

**Comment by unparalleled-js** [comment]
Is the build failing on me?

**Comment by glicht** [comment]
> Is the build failing on me?

No. It is an issue on our side that we are fixing.

**Pull request merged**